app/controllers/bikes_controller.rb

```diff
@@ -0,0 +1,48 @@
+class BikesController < ApplicationController
+  def index
+    @bikes = Bike.new
```

Comment:
here I believe should go a Bike.all, right?

Reply:
Right! done it